lib/opentelemetry/trace/sampling/parent_base_sampler.lua

```diff
@@ -21,7 +21,7 @@ function _M.new(root)
 end
 
 function _M.should_sample(self, parameters)
-    local parent_ctx = parameters.parent_ctx
+    local parent_ctx = parameters.parent_ctx:span_context()
```

Comment:
`parent_ctx` and `parameters.parent_ctx` now have different meaning, `parameters.parent_ctx` should rename to `parameters.context`